numpy/lib/utils.py

```diff
@@ -13,9 +13,86 @@
 __all__ = [
     'issubclass_', 'issubsctype', 'issubdtype', 'deprecate',
     'deprecate_with_doc', 'get_include', 'info', 'source', 'who',
-    'lookfor', 'byte_bounds', 'safe_eval'
+    'lookfor', 'byte_bounds', 'safe_eval', 'show_runtime'
     ]
 
+
+def show_runtime():
+    """
+    Print information about various resources in the system
+    including available intrinsic support and BLAS/LAPACK library
+    in use
+
+    See Also
+    --------
+    show_config : Show libraries in the system on which NumPy was built.
+
+    Notes
+    -----
+    1. Information is derived with the help of `threadpoolctl`
+       library.
+    2. SIMD related information is derived from `__cpu_features__`,
+       `__cpu_baseline__` and `__cpu_dispatch__`
```

Comment:
Double back-ticks to format as code. Single back-ticks for links. I don't think we have links for those attributes.
```suggestion
    2. SIMD related information is derived from ``__cpu_features__``,
       ``__cpu_baseline__`` and ``__cpu_dispatch__``
```

Reply:
Fixed via [3d2fe4e](https://github.com/numpy/numpy/pull/21468/commits/3d2fe4ec796be6ae63c482e4c4f74e3c7867ff1d)